### DISCUSSION TITLE
How can I allow Renovate to completely bypass pull request reviews and then auto merge?

### DISCUSSION THREAD

**Discussion by tal-rofe** [renovatebot/renovate#25919] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I allowed access to Renovate over my GitHub repository, which is public: https://github.com/Exlint/dashboard
My target is to allow Renovate to auto merge pull requests. That is why I configured `renovate.json` file: https://github.com/Exlint/dashboard/blob/main/renovate.json
with `"automerge": true`. But still Renovate cannot auto merge.

And this issue is related with my protected branch, `main`, for sure.
Of course I want best strictness I can do on this branch, but I don't mind allowing Renovate to bypass all the restrictions over this protected branch.

That is why I configured my protected branch as follows:
<img width="884" alt="Screen Shot 2023-11-22 at 22 25 53" src="https://github.com/renovatebot/renovate/assets/100444463/d4753d1b-841b-43f5-a11a-45dfad355c27">
<img width="788" alt="Screen Shot 2023-11-22 at 22 26 43" src="https://github.com/renovatebot/renovate/assets/100444463/a8c75848-1c72-4fa1-ab0b-0ecfae8d4596">

<img width="849" alt="Screen Shot 2023-11-22 at 22 26 59" src="https://github.com/renovatebot/renovate/assets/100444463/6554f5a5-8309-4b24-9747-993d06b3a792">
<img width="808" alt="Screen Shot 2023-11-22 at 22 27 14" src="https://github.com/renovatebot/renovate/assets/100444463/ff818ab8-aaf3-4c05-ba13-bc40f2b49dcf">

I attached all the relevant images. How could I solve it?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
GitHub's branch protections are complex and change regularly. Answering this question the way you ask it would mean that we need to keep an up-to-date reference to the GitHub branch protection feature, which is not feasible.

We have this: https://docs.renovatebot.com/key-concepts/automerge/#frequent-problems-and-how-to-resolve-them

If you encounter a new problem, we can try to help you solve it after looking at logs

**Reply by rarkins** [reply]
To be clear: the question here is about GitHub, not about Renovate specifically - it would apply to any bot or user
